### DISCUSSION TITLE
Authentication to proxy using password

### DISCUSSION THREAD

**Discussion by bropines** [v2rayA/v2rayA#1412] [Ideas]
I probably didn’t find it, but you can make it so that you can’t connect via socks and http without a password, which I can set myself?

**Comment by bropines** [comment]
I understand that no one is fucking up against password authentication? Well, please, well, everything went far and for a long time

**Reply by eronez** [reply]
hook https://v2raya.org/docs/advanced-application/custom-extra-config/
